llvm/test/Analysis/IR2Vec/Inputs/dummy_2D_vocab.json

```diff
@@ -66,7 +66,8 @@
         "ExtractValue": [127, 128],
         "InsertValue": [129, 130],
         "LandingPad": [131, 132],
-        "Freeze": [133, 134]
+        "Freeze": [133, 134],
+        "PtrToAddr": [135, 136]
```

Comment:
@svkeerthy I have no idea what this file does, I just added a new entry for the new instruction. Should it be sorted in enum order instead? I tried to reduce the number of changes to passes I don't understand.